src/app/components/organization-info/index.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import Organization from './organization';
+import * as Constants from '../../constants/constants'
+import { withApollo } from "react-apollo";
+import { loader } from 'graphql.macro';
+import Header from './header';
+import UserInfo from '../user-info';
+
+const GET_ORG_INFO = loader('../../graphql/queries/organization-info.gql');
+const GET_USER_INFO = loader('../../graphql/queries/user-info.gql')
+
+const getRepositories = (client, searchName, query) => {
+  return client.query({
+    query: query,
+    variables: { searchName },
+  });
+};
+
+class HomePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      path: '',
+      filterRepos: [],
+      search: null,
+      errors: null,
+      isOrganization: true,
+      selectedLanguages: [],
+    };
+  }
+
+  componentDidMount() {
+    const searchType = this.props.match.params.typeName;
+    const searchName = this.props.match.params.searchName;
+    if (searchName) {
```

Comment:
@bhoomika08 Can we DRY this method a little?

Reply:
Done. created another function for fetching data and setState. 
Used switch case instead of if/else.